editor/src/components/canvas/ui-jsx.test-utils.tsx

```diff
@@ -322,7 +322,7 @@ export async function renderTestEditorWithModel(
         numberOfCommits++
       }}
     >
-      <FailJestOnCanvasError />
+      {/* <FailJestOnCanvasError /> */}
```

Comment:
I think this is a leftover change from debugging

Reply:
Ah, you were too quick! I was still running the tests locally with this uncommented, fixed in 736376f57.